app/views/teams/_form.html.slim

```diff
@@ -77,6 +77,7 @@
       = f.simple_fields_for :conference_preferences, f.object.conference_preferences.sort{ |a,b| a.option <=> b.option} do |c|
         = c.hidden_field :option, value: c.object.option
         = c.input :conference_id, collection: @conferences.group_by(&:region), as: :grouped_select, group_method: :last, label: c.object.option == 1 ? "Primary choice" : "Secondary Choice", prompt: "Conference", wrapper_html: { class: 'col-md-8' }
+    = link_to 'I want to suggest another conference!', new_conference_path(team: @team.id), class: 'btn btn-primary col-sm-offset-3'
```

Comment:
Ah, I just came to realise that the team parameter is probably only to eventually redirect back to the team's page. Can't we determine the student's team in the controller? This could be somewhat related to #790 (i.e. determining the currently relevant student team if a user is part of more than one team). It's not a blocker for this PR but it would be good if you gave it a try.